tornadox/ivp_examples.py

```diff
@@ -0,0 +1,300 @@
+"""Initial value problem examples."""
```

Comment:
Just wanted to say that I'm a fan of this file. Already looking forward to using that :D

Reply:
... I was even thinking about making this a standalone package :D

Reply:
tbh I would also be fine with that :D